zanata-model/src/main/java/org/zanata/model/HTextFlowTarget.java

```diff
@@ -117,10 +118,25 @@
 
     private List<HTextFlowTargetReviewComment> reviewComments;
 
+    @Getter
+    private String revisionComment;
+
+    private boolean revisionCommentChanged = false;
+
     // Only for internal use (persistence transient)
     @Setter(AccessLevel.PRIVATE)
     private Integer oldVersionNum;
 
+    public void setRevisionComment(String revisionComment) {
+        this.revisionComment = revisionComment;
+        revisionCommentChanged = true;
```

Comment:
Maybe it would be better to call it `revisionCommentSet` then, since that could mean it is set to the same thing, that way there won't be danger of someone else also thinking that it should have a check.